data-upgrade-users/src/main/java/org/exoplatform/migration/UserSetExternalInGateinPortal.java

```diff
@@ -40,6 +42,14 @@ public UserSetExternalInGateinPortal(OrganizationService organizationService,Ini
 
   }
   @Override
+  public boolean shouldProceedToUpgrade(String newVersion,
```

Comment:
why do we need that @sofyenne ?

Reply:
To ensure that the upgrade plugin executes only once!